helpers/freedom_helper.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+const cudl = require('cuddle');
```

Comment:
@pandafox  @wilbertverayin Should we use axios or fetch api? I think fetch is pretty stable ?

Reply:
@pandafox @wilbertverayin @wildtron Hi guys, any follow-up on this? Thanks!

Reply:
Ah right2, let's switch this to axios already

Reply:
Acknowledged. Thanks for the help guys.

Reply:
@cedricabuso @wildtron The `axios` have the same functionality as `cuddle` right?

Reply:
Yup, just different way of doing things

Reply:
@wildtron @cedricabuso I changed this now from `cuddle` to `Axios`